docs/support-bundle-guide.md

```diff
@@ -0,0 +1,183 @@
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default
+```
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-vm-external-node
+spec:
+  externalNodes:
+    nodeNames: // empty list means all the ExternalNodes in namespace
+      - vm
+    namespace: vm-ns
+  fileServer:
+    url: yourtestdomain.com:22/root/test //scheme sftp can be omitted
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default
+```
+
+You can check the status of SupportBundleCollection by running
+`kubectl get supportbundlecollections NameOfSupportBundleCollection -ojson`. The following example
+shows a successful creation of a SupportBundleCollection. `collectedNodes` and `desiredNodes`
+should match the number of Nodes/ExternalNodes you want to collect support bundle. There is also
+an entry of "Completed" status in the "conditions" list.
+
+```bash
+$ kubectl get supportbundlecollections collect-support-bundle-for-kube-nodes -ojson
+
+...
+ "status": {
+        "collectedNodes": 1,
+        "conditions": [
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "True",
+                "type": "Started"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "BundleCollected"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "False",
+                "type": "CollectionFailure"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "Completed"
+            }
+        ],
+        "desiredNodes": 1
+    }
+```
+
+To access the files, SSH into yourtestdomain.com with port 22. Three tarballs,
+two for `collect-support-bundle-for-kube-nodes` and one for
+`collect-support-bundle-for-vm-external-node` are expected to be shown in `/root/test` folder.
+Run the `tar xvf TARBALL_NAME` command and you will get the files,
+each with different collect items.
+
+## List of collected items
+
+Depending on the methods you use to collect support bundle, the items you collect may differ.
+Here is the table that shows such differences.
+
+We use `A`,`C` and `O` to represent "running `antctl supportbundle` in the Antrea Agent",
+"running `antctl supportbundle` in the Antrea Controller" and
+"running `antctl supportbundle` out-of-cluster". Also, We use `SBC-N` and `SBC-E` to represent
+"apply SupportBundleCollection CRD to create support bundle on Nodes" and
+"apply SupportBundleCollection CRD to create support bundle on ExternalNodes".
+
+Please refer to the corresponding [antctl user guide section](antctl.md) for the detail of `antctl` commands.
```

Comment:
I don't get this, should this be removed?

Reply:
I mean there are lots of antctl commands in the following table, which may confuse the readers. He/She can check the antctl document for detail